### ISSUE TITLE
Hyperparameter Tuning

### ISSUE THREAD

**Issue by carlo-scr** [carlo-scr/Beyond-Sentiment-Multi-label-Emotion-Detection-in-Text#11]
Use Optuna for better control fine-tuning models.
